### ISSUE TITLE
Program remains running after closing result window

### ISSUE THREAD

**Issue by severinstojanovic** [skeeto/vulkan-test#2]
Nifty little program!

> "Successfully created a Vulkan instance."
> Close program.
> Task Manager -> still running -> have to manually end the process.

Windows 10.

**Comment by skeeto** [comment]
Yeah, it seems like I really need to debug this on a variety of Windows 
10 systems. I don't have any real Windows 10 machines around right now, 
and it works fine on the virtual systems (with no Vulkan at all).
